### DISCUSSION TITLE
New Datas acc Servicecloud [Dec 2024]

### DISCUSSION THREAD

**Discussion by MiG-41** [Egyras/HeishaMon#611] [General]
In Servicecloud i can see new positions:
- Electrical Anode (Actual) [ 1:Off 2:On 3:Error]
- Outdoor heat exchanger middle temperature [oC]

Also not sure ,if we have already this:
-SHP control [ 1:Disable 2:Enable]
-SHP flow control( forbid deltaT)[ 1:Disable 2:Enable]

Maybe somone have new HP with this features , and can pass a log ?

EDIT: 
Byte 10:
7th & 8th bit = SHP Control b01 - Disable b10 - Enable
5th & 6th bit = SHP Flow control( forbid ΔT) b01- Disable b10 - Enable

Byte 115:
1st & 2nd bit = DHW Circulation b01 - off b10 - on

Byte 121:
1st & 2nd bit = Electrical Anode :  b01 - off b10 - on b11 - Error

For DHW Circulation and Electrical Anode byte 199 equal or greater then 0x05 ....

**Comment by MiG-41** [comment]


**Reply by geduxas** [reply]
How it can get this?  I don't remember any special contacts for speed feedback..

**Comment by MiG-41** [comment]
It become a little bit fun....

Byte 194 is Outdoor heat exchanger middle temperature .... But only when Byte 199 is greather then 0x03 ( so when extra query exist) , otherwise is a part of power generation...

**Reply by geduxas** [reply]
I suspect byte 199 will be some protocol version.. and why they adding to old bytes if there is plantly available space.. what a mess

**Comment by MiG-41** [comment]
And Byte 193 is Zone 2 pump speed [rpm] - Convert to DEC (X-1) X 50
Ofcourse when Byte 199 is greater then 0x03...

**Comment by MiG-41** [comment]
For DHW Circulation and Electrical Anode byte 199 must be equal or greater then 0x05 ,then service cloud start to interprete this byte 115 and 121  ....
